website/cue/reference/components/sinks/datadog_archives.cue

```diff
@@ -0,0 +1,247 @@
+						required:    false
+						warnings: []
+						type: object: {}
+					}
+					acl:                    sinks.aws_s3.configuration.acl
+					grant_full_control:     sinks.aws_s3.configuration.grant_full_control
+					grant_read:             sinks.aws_s3.configuration.grant_read
+					grant_read_acp:         sinks.aws_s3.configuration.grant_read_acp
+					grant_write_acp:        sinks.aws_s3.configuration.grant_write_acp
+					server_side_encryption: sinks.aws_s3.configuration.server_side_encryption
+					ssekms_key_id:          sinks.aws_s3.configuration.ssekms_key_id
+					storage_class: {
+						category: "Storage"
+						common:   false
+						description: """
+          			The storage class for the created objects. See [the S3 Storage Classes](https://docs.aws.amazon.com/AmazonS3/latest/dev/storage-class-intro.html) for more details.
+          			Log Rehydration supports all storage classes except for Glacier and Glacier Deep Archive.
+          			"""
+						required: false
+						warnings: []
+						type: string: {
+							default: null
+							enum: {
+								STANDARD:            "The default storage class. If you don't specify the storage class when you upload an object, Amazon S3 assigns the STANDARD storage class."
+								REDUCED_REDUNDANCY:  "Designed for noncritical, reproducible data that can be stored with less redundancy than the STANDARD storage class. AWS recommends that you not use this storage class. The STANDARD storage class is more cost effective. "
+								INTELLIGENT_TIERING: "Stores objects in two access tiers: one tier that is optimized for frequent access and another lower-cost tier that is optimized for infrequently accessed data."
+								STANDARD_IA:         "Amazon S3 stores the object data redundantly across multiple geographically separated Availability Zones (similar to the STANDARD storage class)."
+								ONEZONE_IA:          "Amazon S3 stores the object data in only one Availability Zone."
+							}
+							syntax: "literal"
+						}
+					}
+					tags: sinks.aws_s3.configuration.tags
+					region: {
+						description: "The [AWS region](\(urls.aws_regions)) of the target service."
+						required:    true
+						type: string: {
+							examples: ["us-east-1"]
+							syntax: "literal"
+						}
+					}
+				}
+			}
+		}
+	}
+
+	input: {
+		logs:    true
+		metrics: null
+	}
+
+	how_it_works: {
+		a_general: {
+			title: "General overview"
+			body: """
+				The sink is implemented as a simple wrapper over corresponding storage service sinks,
+				with the following customizations for compatibility with DataDog Log Rehydration:
+				- custom object key format
+				- event pre-processing
+				- batch parameters, optimized for Log Rehydration
+				- events are always JSON-encoded
+				- batches are always gzip-compressed
+
+				Therefore some configuration parameters are not exposed to users - parameters that impact object names,
+				batch parameters, compression or encoding format.
```

Comment:
Take it or leave it: I'd likely leave out the list of implementation specifics here.  It's not pertinent to users; we really only care about getting the point across that this sink is compatible with Datadog log rehydration.

Reply:
yeah, I agree with you, I'll remove it.

Reply:
done.